storage/engine/mvcc.go

```diff
@@ -1227,6 +1250,38 @@ func MVCCConditionalPut(
 	iter := engine.NewIterator(key)
 	defer iter.Close()
 
+	return mvccConditionalPutUsingIter(ctx, engine, iter, ms, key, timestamp, value, expVal, txn)
+}
+
+// MVCCBlindConditionalPut is a fast-path of MVCCConditionalPut. See the
+// MVCCConditionalPut comments for details of the
+// semantics. MVCCBlindConditionalPut skips retrieving the existing metadata
+// for the key requiring the caller to guarantee no versions for the key
+// currently exist.
+func MVCCBlindConditionalPut(
```

Comment:
Having this method doesn't really make sense because you should just call `MVCCBlindPut` directly for a `ConditionalPut` command which expects the entry does not yet exist and where you determine it in fact does not exist. Creating this for parity just seems wasteful.